scripts/validate-artifacts.ps1

```diff
@@ -1,121 +1,75 @@
-
-
-Write-Output "Validating artifacts"
-
-$data = Import-Excel -StartRow 3 -Path "$tempFolderPath/$spreadsheetName" -WorksheetName $worksheetName
-
-$computeEstimate = $data | Where-Object {$_.'Service Category' -eq 'Compute'}
-if ($computeEstimate) { 
-    if ($computeEstimate.'Service Category'.Count -eq 1) { 
-        if ($computeEstimate.'Region' -ne 'UK West') { 
-            throw "Unable to verify the VM region - please make sure it set to UK West and try again."
-        } 
-
-        if (-not $computeEstimate.Description.Contains("B1s")) { 
-            throw "Unable to verify VM size in calulations - please make sure that VM size is set to B1s in the calculations and try again. "
-        } 
-        
-        if (-not $computeEstimate.Description.Contains("Linux")) { 
-            throw "Unable to verify VM OS type in calulations - please make sure that VM OS tyoe is set to Linux in the calculations and try again. "
-        } 
-
-        if (-not $computeEstimate.Description.Contains("1 managed disk – P4")) { 
-            throw "Unable to verify that you included OS disk to the calulations - please check and try again. "
-        } 
-        
-        Write-Output "`u{2705} Checked calculations for the Virtual Machine - OK."
-
-    } else { 
-        throw "Unable to verify estimates for the category 'Compute' - more than one VM found. Please make sure that you have only one VM in your calculations and try again."
-    }
-
+Write-Output "Checking if artifacts storage container exists" 
+$artifactContainer = Get-AzStorageContainer -Name $containerName -Context $storageAccount.Context -ErrorAction SilentlyContinue
+if ($artifactContainer) { 
+    Write-Output "Storage container for artifacts found!" 
 } else { 
-    throw "Unable to find estimates for the category 'Compute'. Please make sure that you added your Virtual Machine to the pricing calculation."
+    throw "Unable to find a storage container $containerName in the storage account $ArtifactsStorageAccountName, please make sure that it's created"
 }
 
-$storageEstimate = $data | Where-Object {$_.'Service Category' -eq 'Storage'}
-if ($storageEstimate) { 
-    if ($storageEstimate.'Service Category'.Count -eq 1) { 
-        if ($storageEstimate.'Region' -ne 'UK West') { 
-            throw "Unable to verify the VM region - please make sure it set to UK West and try again."
-        } 
-
-        if (-not $storageEstimate.Description.Contains("Managed Disks, Premium SSD, LRS Redundancy")) { 
-            throw "Unable to verify disk type - please make sure that you added a mannaded disk with proper disk type (Premium SSD) and replication type (LRS) to the calculations and try again."
-        } 
-        
-        if (-not $storageEstimate.Description.Contains("P6")) { 
-            throw "Unable to verify data disk size - please make sure that disk size for data disk is set to 64GB and try again. "
-        } 
+# generation of artifacts
+Write-Output "Generating artifacts"
```

Comment:
The section labeled 'generation of artifacts' suggests that this script is generating artifacts rather than validating them. If the script's purpose is to validate, consider revising this section to focus on validation tasks instead.